lib/patreon/oauth.rb

```diff
@@ -30,9 +30,10 @@ def refresh_token(refresh_token, redirect_uri)
     private
 
     def update_token(params)
-      url = URI.parse('https://api.patreon.com/oauth2/token')
+      url = URI.parse('https://www.patreon.com/api/oauth2/token')
```

Comment:
It seems a little odd that we construct a client in `api.rb` but don't use it here

Reply:
There's lots of odd here, would you prefer we also construct a client here?

Reply:
I'll submit a pr after this to make consolidate the client construction, will be way easier to illustrate there :)